functions/gmail-notifier/index.js

```diff
@@ -76,6 +77,18 @@ exports.initWatch = async (req, res) => {
   return res.send(`Watch initialized on gmail inbox: ${email}`)
 }
 
+exports.postMessageToSlack = async (text) => {
+  const token = 'slack-bot-token'
+  const web = new WebClient(token)
+  const conversationID = 'C021BGBSMFY'
+  try {
+    const res = await web.chat.postMessage({ channel: conversationID, text })
+  } catch (err) {
+    console.error(err)
```

Comment:
is this alright in nodejs ? or should be throwing error ?

Reply:
It's alright imho, that's the canonic way to catch errors when using the await syntax, it's the equivalent to ".catch(" after a ".then(".